workflow/envs/vg.yaml

```
@@ -0,0 +1,5 @@
+name: vg
+channels:
+   - bioconda
+dependencies:
+   - vg
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)